.github/workflows/ci.yml

```diff
@@ -69,24 +69,27 @@ jobs:
           - preset: "vc6-profile"
             tools: true
             extras: true
-          - preset: "vc6-internal"
-            tools: true
-            extras: true
           - preset: "vc6-debug"
             tools: true
             extras: true
-          - preset: "win32-vcpkg"
-            tools: true
-            extras: true
-          - preset: "win32-vcpkg-profile"
+          - preset: "win32"
             tools: true
             extras: true
-          - preset: "win32-vcpkg-internal"
+          - preset: "win32-profile"
             tools: true
             extras: true
-          - preset: "win32-vcpkg-debug"
+          - preset: "win32-debug"
             tools: true
             extras: true
+          # - preset: "win32-vcpkg"
```

Comment:
Please explain in a comment why this is disabled right now for the next maintainer of this